core-bundle/src/Resources/contao/elements/ContentVimeo.php

```diff
@@ -97,6 +101,23 @@ protected function compile()
 			$url .= '#t=' . (int) $this->playerStart . 's';
 		}
 
+		if ($this->videoSplash)
+		{
+			$objFile = FilesModel::findByUuid($this->singleSRC);
+
+			if ($objFile !== null && \is_file(TL_ROOT . '/' . $objFile->path))
+			{
+				$this->singleSRC = $objFile->path;
+				$objSplash = new \stdClass();
+				$this->addImageToTemplate($objSplash, $this->arrData, null, null, $objFile);
+				$this->Template->splashImage = $objSplash;
+			}
+			else
+			{
+				$this->videoSplash = false;
```

Comment:
I changed it now. I can't make it the same way as in the `ContentText`, since the `Controller::addImageToTemplate` sets `addImage` of the given template to `true` by itself (and in our case the variable is called `videoSplash`).